youtube2zim/scraper.py

```diff
@@ -244,6 +244,9 @@ def run(self):
         # validate dateafter input
         self.validate_dateafter_input()
 
+        # validate youtube_id input
+        self.validate_id()
```

Comment:
I'd move that before the dateafter one as this is the main user input so makes more sens to test that first